feature/community/src/main/java/com/iteneum/community/presentation/CommunityComposable.kt

```diff
@@ -23,10 +23,10 @@ import com.iteneum.designsystem.theme.LeasePertTheme
  *this view model is used to handle the loading of the components of the interface
  * including icons, text and button click functionality.
  *
- * @author Andrés Ivan Medina Herrera
+ * @author Andres Ivan Medina Herrera
  */
 @Composable
-fun CommunitylistUI() {
+fun CommunityListUI() {
```

Comment:
Just change the view name to **CommunityView**, since there is a screen called **CommunityListView** and it could cause confusion.

![image](https://user-images.githubusercontent.com/42255248/235195259-fded2105-99c5-4d74-8b72-f2659405e998.png)

Reply:
gotcha